geo_autoRIFT/geogrid/src/geogridOptical.cpp

```diff
@@ -1065,6 +1102,10 @@ void geoGridOptical::geogridOptical()
                         raster2b[jj] = nodata_out;
                     }
                     
+                    // range/azimuth distance to DEM distance scale factor
+                    sf_raster1[jj] = norm_C(xdiff) / std::abs(XSize);
+                    sf_raster2[jj] = norm_C(ydiff) / std::abs(YSize);
```

Comment:
OK, so in this case we now have scale factors but raster1a (1b,2a,2b) remain unmodified. So i guess the scale is still removed from the transformation matrix but now we need to multiply the rotation matrix by the scale factors (sf_raster1 (2)) when transforming the data. OK, I see the addition in netcdf_output.py

Reply:
@alex-s-gardner you are right about the workflow.